Items/04_06_MachineGuns_HeavyWeps_&_RocketLaunchers.txt

```diff
@@ -94,6 +94,32 @@ Movement Speed Penalty: -2m
 * 4 out to 45m
 * 5 out to 80m
 
+==Frontier Flame Thrower FLMR-B $3880 ==
+
+Damage: 80 Special

```

Comment:
20 less damage than the level 1 flame thrower? I suppose there's the potential to deal it to multiple targets, so maybe that makes sense. Can we change it to deal fire damage? https://github.com/bleehu/Compound_X/blob/play/01_04_Damage_Types_and_Armor_Types.txt#L51

Reply:
I see the original flame thrower is special damage. Should that also be fire damage?